src/test/resources/unit/spdx-maven-plugin-test/json-pom-dependencies.xml

```diff
@@ -0,0 +1,107 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.spdx</groupId>
+    <artifactId>spdx-maven-plugin-test</artifactId>
```

Comment:
I had to replace the whitespaces for this test case because it threw the following error when building the MavenProject using `ProjectBuilder`:
```
org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
[ERROR] 'artifactId' with value 'spdx maven plugin test' does not match a valid id pattern. @ line 7, column 17
```